inspire/base/templates/format/record/Inspire_Default_HTML_brief.tpl

```diff
@@ -30,21 +30,25 @@
       <div class="col-md-9"  id="left-column">
         <h4 class="custom-h">
           <b>
+            {% set title_list = [] %}
             {% if record.titles %}
               {% for title in record['titles'] %}
+                {% do title_list.append(title['title']) %}
```

Comment:
I am not sure if we are doing too much here? Now with an stable data model `record['titles']` should always be a list.

Can you check if `record['titles[0].title']` works? Note that this special notation is possible because the `record` object is a `SmartDict` from Invenio.